gdal_subsetter/utilities.py

```diff
@@ -71,45 +70,46 @@ def is_geotiff(file_name: str) -> bool:
     return gdalinfo_lines[0] == 'Driver: GTiff/GeoTIFF'
 
 
```

Comment:
Main reason for changes here: because netCDF4 files won't be extracted, this function can focus on GeoTIFFs. Then when I was looking at it, I wanted to make the formatting a bit more like we've adopted elsewhere and simplify (main other simplification is identification of files from the extracted zip file with a GeoTIFF suffix).